google/cloud/aiplatform/_featurestores/feature.py

```diff
@@ -0,0 +1,306 @@
+from google.cloud.aiplatform import utils
+from google.cloud.aiplatform.utils import featurestore_utils
+
+_LOGGER = base.Logger(__name__)
+
+
+class Feature(base.VertexAiResourceNounWithFutureManager):
+    """Managed feature resource for Vertex AI."""
+
+    client_class = utils.FeaturestoreClientWithOverride
+
+    _is_client_prediction_client = False
+    _resource_noun = None
+    _getter_method = "get_feature"
+    _list_method = "list_features"
+    _delete_method = "delete_feature"
+
+    def __init__(
+        self,
+        feature_name: str,
+        featurestore_id: Optional[str] = None,
+        entity_type_id: Optional[str] = None,
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+    ):
+        """Retrieves an existing managed feature given a feature resource name or a feature ID.
+
+        Example Usage:
+
+            my_feature = aiplatform.Feature(
+                feature_name='projects/123/locations/us-central1/featurestores/my_featurestore_id/\
+                entityTypes/my_entity_type_id/features/my_feature_id'
+            )
+            or
+            my_feature = aiplatform.Feature(
+                feature_name='my_feature_id',
+                featurestore_id='my_featurestore_id',
+                entity_type_id='my_entity_type_id',
+            )
+
+        Args:
+            feature_name (str):
+                Required. A fully-qualified feature resource name or a feature ID.
+                Example: "projects/123/locations/us-central1/featurestores/my_featurestore_id/entityTypes/my_entity_type_id/features/my_feature_id"
+                or "my_feature_id" when project and location are initialized or passed, with featurestore_id and entity_type_id passed.
+            featurestore_id (str):
+                Optional. Featurestore ID to retrieve feature from, when feature_name is passed as Feature ID.
+            entity_type_id (str):
+                Optional. EntityType ID to retrieve feature from, when feature_name is passed as Feature ID.
+            project (str):
+                Optional. Project to retrieve feature from. If not set, project
+                set in aiplatform.init will be used.
+            location (str):
+                Optional. Location to retrieve feature from. If not set, location
+                set in aiplatform.init will be used.
+            credentials (auth_credentials.Credentials):
+                Optional. Custom credentials to use to retrieve this Feature. Overrides
+                credentials set in aiplatform.init.
+        """
+        (
+            featurestore_id,
+            entity_type_id,
+            _,
+        ) = featurestore_utils.validate_and_get_feature_resource_ids(
+            feature_name=feature_name,
+            entity_type_id=entity_type_id,
+            featurestore_id=featurestore_id,
+        )
+
+        self._resource_noun = (
```

Comment:
Add a comment this this is a workaround until the base class supports nested features.